### DISCUSSION TITLE
Welcome to AgentBench Discussions!

### DISCUSSION THREAD

**Discussion by Xiao9905** [THUDM/AgentBench#131] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ptrrrr** [comment]
Hi.  I am looking at evaluating an custom agent framework.  Any ideas on how I can go about this.  This will be a framework and not an LLM.

**Comment by zhc7** [comment]
Hi, @ptrrrr maybe #118 helps? As long as the agent framework is able to implement the interface in `src/client/agent.py`, you may treat it as an agent.
